siti_tools/siti.py

```diff
@@ -115,13 +115,54 @@ def ti(
         return (frame_data - previous_frame_data).std()
 
 
-def eotf_1886(frame_data: np.ndarray, gamma: float = DEFAULT_GAMMA) -> np.ndarray:
-    # TODO implement
+def eotf_1886(frame_data: np.ndarray, gamma: float = DEFAULT_GAMMA, l_min: float = 0.0,
+              l_max: float = 1.0) -> np.ndarray:
+    """
+    Electro-optical transfer function defined in ITU-T Rec. BT.1886
+
+    Args:
+        frame_data: pixel values expected in the range [0, 1]
```

Comment:
The pixel values passed to this function will be coming from `apply_display_model`, and they are not in the range `[0-1]`, but whatever the input is using (e.g. 0-255 for 8-bit, etc.)

How should we normalize these pixel values if we don't know the bit depth? Does the user need to specify it manually?